polyfill/lib/calendar.mjs

```diff
@@ -1322,6 +1322,14 @@ const helperIslamic = ObjectAssign({}, nonIsoHelperBase, {
   DAYS_PER_ISLAMIC_YEAR: 354 + 11 / 30,
   DAYS_PER_ISO_YEAR: 365.2425,
   constantEra: 'ah',
+  reviseIntlEra(calendarDate /*, isoDate*/) {
+    let { era, eraYear } = calendarDate;
+    // Chrome for Android as of v 142.0.6367.179 mishandled the era option in Intl.DateTimeFormat
+    // and returned 'bc' instead of 'ah'. This code corrects that and any possible future errors.
+    // see https://issues.chromium.org/issues/40856332
+    era = 'ah';
+    return { era, eraYear };
```

Comment:
Let's simplify this a bit and reduce bundle size of the production polyfill. 

```suggestion
    // Chrome for Android as of v 142.0.6367.179 mishandled the era option in Intl.DateTimeFormat
    // and returned 'bc' instead of 'ah'. This code corrects that and any possible future errors.
    // See https://issues.chromium.org/issues/40856332
    return { ...calendarDate, era: 'ah' };
```

Reply:
That's a cleaner approach indeed. I have made the changes